### DISCUSSION TITLE
Atlantis Upgrade 15.0.0 to 16.0.0

### DISCUSSION THREAD

**Discussion by bp529** [primefaces/community#150] [PrimeNG Templates] [answered]
Are there any instructions on how to upgrade an existing project to the latest release that includes the "Remastered" Atlantis updates?

**Comment by jncalderon** [comment]
Hi, this worked for me, it was a different theme (Mirage), might works for you:

https://github.com/orgs/primefaces/discussions/130#discussioncomment-7130755

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

> Hi, this worked for me, it was a different theme (Mirage), might works for you:
> 
> https://github.com/orgs/primefaces/discussions/130#discussioncomment-7130755

The provided solution above is for updating PrimeNG and sass-theme. It does not update the template to v16, it just updates the PrimeNG version and primeng-sass-theme. It doesn't fix or update the layout or possible bugs related to template. The reason it worked with @jncalderon is the issue caused by theming, not related to the template itself. For the template versions, we suggest you wait for official template updates. 

To make it clear, yes you can use the guidance and update PrimeNG & sass-theme in your template. But please note that official template updates may introduce new features, template-related bug fixes, or enhancements, and the solution provided above does not cover that. For the new features, enhancements, or layout/logic-related fixes you need to wait for the official Atlantis v16 update.
